superx/information_extractor.py

```diff
@@ -0,0 +1,131 @@
+            'http://prices.shufersal.co.il/FileObject/UpdateCategory?catID=2&storeId=0&sort=Category&sortdir=ASC&page=12',
+            'http://prices.shufersal.co.il/FileObject/UpdateCategory?catID=2&storeId=0&sort=Category&sortdir=ASC&page=13',
+            'http://prices.shufersal.co.il/FileObject/UpdateCategory?catID=2&storeId=0&sort=Category&sortdir=ASC&page=14',
+            'http://prices.shufersal.co.il/FileObject/UpdateCategory?catID=2&storeId=0&sort=Category&sortdir=ASC&page=15',
+            'http://prices.shufersal.co.il/FileObject/UpdateCategory?catID=2&storeId=0&sort=Category&sortdir=ASC&page=16',
+            'http://prices.shufersal.co.il/FileObject/UpdateCategory?catID=2&storeId=0&sort=Category&sortdir=ASC&page=17',
+            'http://prices.shufersal.co.il/FileObject/UpdateCategory?catID=2&storeId=0&sort=Category&sortdir=ASC&page=18',
+            'http://prices.shufersal.co.il/FileObject/UpdateCategory?catID=2&storeId=0&sort=Category&sortdir=ASC&page=19',
+            'http://prices.shufersal.co.il/FileObject/UpdateCategory?catID=2&storeId=0&sort=Category&sortdir=ASC&page=20',
+            'http://prices.shufersal.co.il/FileObject/UpdateCategory?catID=2&storeId=0&sort=Category&sortdir=ASC&page=21',
+            'http://prices.shufersal.co.il/FileObject/UpdateCategory?catID=2&storeId=0&sort=Category&sortdir=ASC&page=22',  # shufersal
+            'http://matrixcatalog.co.il/NBCompetitionRegulations.aspx'      # victory
+            ]
+
+# current supermarket name
+super_name = ''
+# connects to relevant sql database
+conn = mysql.connector.connect(user='root', password='', host='localhost', database='testdb')
+# cursor for db
+cursor = conn.cursor()
+
+
+def get_zip_file_links():
+    """
+    web scrapes the urls in url_list and gets the links for the xml files
+    :return: a list of links to the zip files
+    """
+    for url in url_list:
+        page = requests.get(url)
+        web_scrapper = BeautifulSoup(page.content, 'html.parser')
+        links_list = web_scrapper.find_all('a')
+        zip_links = set()
+
+        for link in links_list:
+            if link.has_attr('href'):
+                https = str(link.attrs['href'])
+                if 'PriceFull' in https:
+                    zip_links.add(https)
+
+        global super_name
+        if 'mega' in url:
+            super_name = 'mega'
+        elif 'shufersal' in url:
+            super_name = 'shufersal'
+        elif 'matrix' in url:
+            super_name = 'victory'
+
+        info_parser(zip_links)
+
+
+def info_parser(zip_links):
+    """
+    parses info for supermarket
+    :param zip_links: list of zip file links from the website
+    """
+    i = 0
+    for zip_link in zip_links:
+        # corrects the urls
+        if super_name == 'mega':
+            zip_link = url_list[0] + '/' + zip_link
+        elif super_name == 'victory':
+            zip_link = 'http://matrixcatalog.co.il/' + zip_link
```

Comment:
in your url list, victory is listed as: http://matrixcatalog.co.il/NBCompetitionRegulations.aspx
This code will add "http://matrixcatalog.co.il/" to the front of that.

Reply:
i think you misunderstood this, the zip_link is not the link in the url, it is the link of the zip file online. the pages HTML does only has the suffix of the link so it is necessary to add the proper prefix

Reply:
definitely misunderstood